frontend/src/components/SchemaList.vue

```diff
@@ -1,23 +1,86 @@
 <template>
-  <ul class="list-group sticky-top">
-    <li
-      v-on:click="$emit('selectSchema', schema)"
-      v-for="schema in schemas"
-      :key="schema.id"
-      class="list-group-item"
-      :class="selectedSchema === schema ? 'active' : ''"
-    >
-      {{ schema.name }}
-    </li>
-  </ul>
+  <li class="nav-item dropdown">
+    <a class="nav-link dropdown-toggle" href="#" id="nav-schema-dropdown" role="button"
+       data-bs-toggle="dropdown" aria-expanded="false">
+      {{ title }}
+    </a>
+    <ul class="dropdown-menu" aria-labelledby="nav-schema-dropdown">
+      <li v-if="loading">
+        <div class="dropdown-item placeholder-wave">
+          <span class="placeholder col-8"></span>
+        </div>
+      </li>
+      <li v-for="schema in schemas" :key="schema.id">
+        <button type="button" class="dropdown-item" :class="modelValue === schema ? 'active': ''"
+           v-on:click="selectSchema" :value="schema.id">
+          {{ schema.name }}
+        </button>
+      </li>
+      <li>
+        <hr class="dropdown-divider">
+      </li>
+      <li>
+        <RouterLink
+            to="/createSchema"
+            class="dropdown-item">
+          <i class='eos-icons'>add_circle</i>
+          New
+        </RouterLink>
+      </li>
+    </ul>
+  </li>
 </template>
 
 <script>
+import {api} from "@/api";
+
 export default {
   name: "SchemaList",
-  props: ["schemas", "selectedSchema"],
-  emits: ["selectSchema"],
-  computed: {},
-  methods: {},
+  props: ["modelValue"],
+  emits: ["update:modelValue"],
+  data: function () {
+    return {
+      schemas: null,
+      loading: true
+    }
+  },
+
+  components: {},
+  computed: {
+    title() {
+      if (this.modelValue !== undefined && this.modelValue !== null) {
+        return `Schema: ${this.modelValue.name}`;
+      }
+      return "Schema";
+    }
+  },
+  mounted: function () {
```

Comment:
Probably we should have `created` here instead of `mounted`🤔

Reply:
That should also work, yes :slightly_smiling_face:

Reply:
Changed it.